libs/full/compute_cuda/tests/unit/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ set(default_executor_CUDA On)
 set(for_each_compute_CUDA On)
 set(for_loop_compute_CUDA On)
 
-if(HPX_WITH_CUDA_CLANG OR HPX_WITH_HIP)
+if(HPX_WITH_CLANG_CUDA OR HPX_WITH_HIP)
```

Comment:
By the way, any particular reason for renaming `HPX_WITH_CUDA_CLANG` to `HPX_WITH_CLANG_CUDA`? Is its purpose only to internally signal that we have CUDA enabled with clang, or are users still expected to set it?

Reply:
The variable is not in the cache anymore so it is only internal, no user will have to set it, and I just renamed it as the environment is called  `env-clang-cuda` and it's the most intuitive name, I think..
But if anyone is opposed to that I can revert the change. I figured it was impacting nobody ^^'

Reply:
Sounds good, no objections.